src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/ReducibleAnnotationData.java

```diff
@@ -1,6 +1,7 @@
 package org.broadinstitute.hellbender.tools.walkers.annotator.allelespecific;
```

Comment:
We may eventually need to introduce something like an `AnnotationData` supertype for `ReducibleAnnotationData` to eliminate these `@SuppressWarnings`/`instanceof` checks/typecasts that are everywhere in this branch -- but we can address this in a future branch.